src/common/common-domain/src/main/resources/globalconf/v3/shared-parameters.xsd

```diff
@@ -184,6 +184,22 @@
                     </documentation>
                 </annotation>
             </element>
+            <element name="authenticationCertificateProfileId" type="string" minOccurs="0">
```

Comment:
Can security server 7.4.x download this configuration if new elements are added?